src/fastoad/notebooks/02_explanation/01_summary.ipynb

```diff
@@ -0,0 +1,229 @@
+{
```

Comment:
<blockquote>c [...] the ratio between the wing profile drag and its <strong>wet </strong>area</blockquote><p>Shouldn't it be the <strong>wing </strong>area ? (which is bout half the wet area)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fast-aircraft-design/FAST-OAD/pull/426/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
My idea behind this formula for the computation of the profile drag was to "scale" the formula we use in FAST-OAD-CS23 and FAST-OAD-CS25 which is more or less

$$C_{d0,wing} = f_f * c_f * S_{wet} / S_{ref}$$

with $f_f$, a form factor and $c_f$ a friction coefficient. Though these two depend on the area, I kept them constant as a first approximation. In addition to the fact that the reference area is the wing area and the wet area is proportional to the wing area this makes $C_{d0,wing}$ a constant. 

In the end, I think the best way to go around this is to just not say how it is computed and say that it account for wing drag. I'll also change the notations.

Reply:
Changed in 863446b0